src/main/java/picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigar.java

```diff
@@ -73,7 +74,7 @@
     // Since we inherit from SimpleMarkDuplicatesWithMateCigar, it is useful for us to also inherit the tests
     // which do not contain UMIs.  By default, we don't allow for missing UMIs, but for the inherited tests
     // we allow for missing UMIs.
-    @Option(doc = "Allow for missing UMIs if data doesn't have UMIs.  This option is intended to be used only for testing the code.  Use SimpleMarkDuplicatesWithMateCigar if data has missing UMIs.", optional = true)
+    @Option(doc = "Allow for missing UMIs if data doesn't have UMIs.  This option is intended to be used ONLY for testing the code.  Use SimpleMarkDuplicatesWithMateCigar if data has missing UMIs.", optional = true)
```

Comment:
Could we modify this to say something like?

Use MarkDuplicatesWithMateCigar if data has no UMIs.  Data that contains UMIs where some are missing is not supported.